charts/area-chart/step1.md

```diff
@@ -3,8 +3,23 @@ This provides a collection of React based components you can use to build Patter
 
 PatternFly React charts are composed using separate components, so features are more easily customizable. 
 In this course, we're going to build a PatternFly area chart together - starting with a simple chart, adding multiple datasets, tooltips, axis labels, a legend, and concluding by changing the theme color.
-You'll learn how to use PatternFly React chart components together to build a consistent user experience.
+

```

Comment:
You moved the wrong introduction to the `intro.md` file.... this is the introduction that explains what the tutorial is, the other introduction that you moved is to relevant to the first step. Be sure to read the content that you're moving

Reply:
understood